.github/workflows/build_deploy_prod.yml

```diff
@@ -114,9 +114,31 @@ jobs:
           ref: main
           client_payload: '{"image-tag-prefix": "${{ env.CI_REF_NAME }}", "deploy-env": "prod"}'
 
+  trigger_dd_extension_release:
+    # Trigger private repo to deploy to prod env
+    runs-on: mdb-dev
+    needs: docker_build
+    if: github.actor != 'mindsdbadmin'
+    environment:
+      name: prod
+    # We only want to run one deploy job for an env at a time
+    # Don't cancel in progress jobs because it may be for a different PR
+    concurrency:
+      group: deploy-prod
+      cancel-in-progress: false
+    steps:
+      - uses: FranzDiebold/github-env-vars-action@v2
+      - uses: convictional/trigger-workflow-and-wait@v1.6.5
+        with:
+          owner: mindsdb
+          repo: mindsdb-docker-extension
+          github_token: ${{ secrets.DD_EXT_REPO_DISPATCH_PAT_TOKEN }}
```

Comment:
Can we use `REPO_DISPATCH_PAT_TOKEN` here? I have updated `REPO_DISPATCH_PAT_TOKEN` to allow access to the docker extension repo

Reply:
Done.